data-access.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: article
+titles:
+    # @start locale config
+    en      : &EN       Accessing Data in SDA
+    # @end locale config
+key: page-single
+---
+
+This page contains the information needed in order to request access to data
+submitted to SDA.
+
+## How to request a dataset
+- Apply for a dataset
```

Comment:
Can we add more information here? as a researcher I would have no idea what to do!

Reply:
Yes, I think I will be add at least something.

Reply:
I have added some new text